src/components/custom/_dhero-dmt5.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import { localize } from 'components/localization'
 import { Flex } from 'components/containers'
 import { QueryImage, Header } from 'components/elements'
 import { Button, LinkButton } from 'components/form'
 import device from 'themes/device'
 import useHandleSignup from 'components/hooks/use-handle-signup'
 
+type DHeroProps = {
+    background?: string | JSX.Element
+    background_alt?: string | JSX.Element
+    background_image_name?: string
+    content?: string | JSX.Element
+    go_to_live_demo?: boolean
+    image_name?: string
+    is_mobile?: string | boolean
+    join_us_for_free?: boolean
+    Logo?: string | JSX.Element
+    title?: string
```

Comment:
We can see in file by path `src/pages/dmt5/index.tsx` that we use only 6 of this properties. Please help to make necessary properties like title, content, etc.